Content.Server/Administration/Logs/Converters/EntityStringRepresentationConverter.cs

```diff
@@ -19,6 +28,38 @@ public override void Write(Utf8JsonWriter writer, EntityStringRepresentation val
         if (value.Session != null)
         {
             writer.WriteString("player", value.Session.UserId.UserId);
+
+            if (value.Session is IPlayerSession playerSession &&
+                _adminManager.IsAdmin(playerSession))
```

Comment:
what I meant is
```suggestion
            if (_adminManager.IsAdmin((IPlayerSession) value.Session))
```

Reply:
I’m pretty sure this will work, have already seen such cast, see https://github.com/space-wizards/space-station-14/blob/aee88e1721feb72263a526c2677bb5df3a04793f/Content.Server/Chat/Systems/ChatSystem.cs#L553